process/sandbox.md

```diff
@@ -0,0 +1,99 @@
+# CNCF Sandbox Guidelines v1.0
+![CNCF Sandbox](https://github.com/cncf/artwork/blob/master/other/cncf-sandbox/horizontal/color/cncf-sandbox-horizontal-color.png)
+
+The CNCF Sandbox is the entry point for early stage projects and has four goals:
+
+* Encourage public visibility of experiments or other early work that can add value to the CNCF mission and build the ingredients of a successful Incubation level project
+* Facilitate alignment with existing projects if (and only if) this is desired
+* Nurture projects (e.g. via CNCF [Service Desk](https://github.com/cncf/servicedesk) requests)
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to CNCF legal, code of conduct and IP Policy requirements
+
+This proposal is to move all Inception level projects to the Sandbox and provide clarity on what Sandbox projects stand for.
+
+## What is the CNCF Sandbox
+
+There’s been a desire within the CNCF TOC and community to provide further clarity around project maturity levels in CNCF.
+
+When we initially created the Inception project level, it was intended to provide an avenue for technically interesting early-stage projects that were beneficial to the cloud-native community.  We are transitioning Inception projects to the Sandbox. Sandbox projects should be early-stage projects that the CNCF TOC believes warrant experimentation. The Sandbox should provide a beneficial, neutral home for such projects, in order to foster collaborative development. We aspire to make the Sandbox the preferred path for early-stage projects to enter the CNCF. More mature projects can continue to jump directly to incubation, but as the cloud-native ecosystem grows, we expect to see proportionally more early-stage projects.
+
+## Early Stage
+
+When we say that Sandbox projects are "early stage" this covers the following examples:
+
+1. New projects that are designed to extend one or more CNCF projects with functionality or interoperability libraries. In the case of Kubernetes, the Sandbox is intended as a home for projects that would previously have [started in the Kubernetes Incubator](https://github.com/kubernetes/community/blob/master/incubator.md).
```

Comment:
What is the relationship between the CNCF's and Kube's approval? By that I mean, can the CNCF approve a project w/o Kube's approval? Can Kube approve a project w/o CNCF's approval?  Or is the Kube approval side of this going away and all new Kube incubator projects are put into the CNCF sandbox and just the TOC's approval is needed?

(non-blocking question and the text can be updated later if necessary)

Reply:
Also, add periods (.) to the other lines to be consistent with this one - or remove it from here.

Reply:
@duglin I believe the intention is to shut down the kubernetes incubator long term, but that's a decision for the kubernetes steering committee cc: @bgrant0607 

Each CNCF project can create its own incubator or sub projects without going through the TOC to approve them. The TOC can step in and prevent any bad behavior though by removing projects, but we haven't had this issue yet.

Reply:
The Kubernetes steering committee recently merged this doc on what they intend to replace Kubernetes Incubator with:

https://github.com/kubernetes/community/blob/master/kubernetes-repositories.md

Since Kubernetes itself still intends to offer a home for projects that previously would have been in Kubernetes Incubator, when would it make sense to go directly to Kubernetes vs. the CNCF Sandbox?

Reply:
@enisoc that would be up for the KSC to decide, I haven't had a conversation with @bgrant0607 on his thoughts here yet.